README.md

```diff
@@ -1,51 +1,61 @@
 # movie_radio
 simple radio page, that will play stream from Icecast server
 
-#####to install project on clean VM:
-(Python related stuff)
+###to install project on clean VM:
+#####Step 1 — Installing Ansible
+Add the Ansible PPA and refresh your system's package index, install the ansible
+by typing the following commands:
 
+```
+$ sudo apt-add-repository ppa:ansible/ansible
+$ sudo apt-get update
+$ sudo apt-get install ansible
+```
+
+#####Step 2 — Configuring Ansible Hosts
+Open the file with root privileges like this:
 
 ```
-1. sudo apt-get install git
-2. sudo apt-get install python-pip
-next line is for working python 3 terminal interpreter
-3. sudo apt-get install libreadline-dev
- SSL related dev libs
-4. sudo apt-get install -y make build-essential libssl-dev zlib1g-dev libbz2-dev libsqlite3-dev
-5. sudo apt-get install build-essential python-dev python-setuptools python-pip python-smbus
-6. sudo apt-get install libffi-dev
-7. Install python >=3.5 (see Install Python Section)
-8. pip install virtualenv
-9. virtualenv -p python3.5 env
-Activate virtual env
-10.source env/bin/activate
-11.pip install -r requirments.txt
+$ sudo vim /etc/ansible/hosts
```

Comment:
btw `vim` is not installed iun clean Ubuntu VM, at least this apllyes for 16.04 LTS. This could be replaced with `vi` :D